python/packages/azurefunctions/agent_framework_azurefunctions/_app.py

```diff
@@ -186,20 +216,32 @@ def __init__(
 
         logger.debug("[AgentFunctionApp] Initialization complete")
 
+    @property
+    def agents(self) -> dict[str, AgentProtocol]:
+        """Returns dict of agent names to agent instances.
+
+        Returns:
+            Dictionary mapping agent names to their AgentProtocol instances.
+        """
+        return {name: metadata.agent for name, metadata in self._agent_metadata.items()}
+
     def add_agent(
         self,
         agent: AgentProtocol,
         callback: AgentResponseCallbackProtocol | None = None,
         enable_http_endpoint: bool | None = None,
+        enable_mcp_tool_endpoint: bool | None = None,
```

Comment:
do we need the "endpoint" suffix for "mcp_tool" ?  Does an mcp tool has a concept of an endpoint?